vita3k/config/include/config/config.h

```diff
@@ -50,7 +50,33 @@
     code(bool, "use-ubo", false, use_ubo)                                                               \
     code(bool, "performance-overlay", false, performance_overlay)                                       \
     code(std::string, "backend-renderer", "OpenGL", backend_renderer)                                   \
-    code(int, "user-id", 0, user_id)
+	code(int, "keyboard-button-select", 229, keyboard_button_select)									\
```

Comment:
~~Each number here represents a `SDL_Scancode_*` constant right? I know I suggested dropping the numbers here by themselves, but maybe importing SDL into config might be worth it? It's just difficult to tell which key you're talking about by `229` or `40` etc.~~

I guess they will have to be using these numbers to configure it anyway. No worries, we can tackle this in another pr I think.